binding-library/python/azure/functions_extensions/kafka/kafka.py

```diff
@@ -92,6 +110,22 @@ def decode(cls, data: meta.Datum, *, trigger_metadata) -> KafkaEvent:
 
         return KafkaEvent(body=body)
 
+    @classmethod
+    def decode_multiple_event(cls, data: meta.Datum, trigger_metadata) -> typing.List[KafkaEvent]:
+        if data.type == 'collection_bytes':
+            parsed_data = data.value.bytes
+        
+        elif data.type == 'collection_string':
+            parsed_data = data.value.string
+
+        events = []
+        for i in range(len(parsed_data)):
```

Comment:
Since the index **i** is never used and we are extracting very few contents, we can just simplified this into
```python
events = [KafkaEvent(body=pd) for pd in parsed_data]
```

Reply:
Very cool! I follow your advice and learedn a new expression.